File: Modules/JetPack/src/main/java/com/example/william/my/jet/fragment/BindFragment.java
package com.example.william.my.jet.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.william.my.jet.databinding.JetFragmentBindBinding;

public class BindFragment extends Fragment {

    private JetFragmentBindBinding binding;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = JetFragmentBindBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    @Override
    public void onDestroyView() {
        binding = null;
        super.onDestroyView();
    }
}

File: Modules/module_kotlin/src/main/kotlin/com/example/william/my/module/kotlin/property/BindFragment.kt
package com.example.william.my.module.kotlin.property

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.example.william.my.module.kotlin.databinding.KotlinActivityKotlinBinding

class BindFragment : Fragment() {

    private var binding: KotlinActivityKotlinBinding? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
        binding = KotlinActivityKotlinBinding.inflate(inflater, container, false)
        return binding!!.root
    }

    override fun onDestroyView() {
        binding = null
        super.onDestroyView()
    }
}